src/views/Project/index.jsx

```diff
@@ -12,6 +12,16 @@ import TasksTable from '../../components/TasksTable';
 import issuesQuery from './issues.graphql';
 import bugsQuery from './bugs.graphql';
 
+const componentsFromProduct = product => {
```

Comment:
Can we change this to `componentsFromProducts` and accept a list of products instead of 1 product? This would remove the need to have many `.filter` calls in the file.

Reply:
Sorry, I was thinking something like:

```jsx
const productsList = (products) =>
  products.filter(product => typeof product === 'string');


// search query
products:
  productsList(projects[project].products).length
    ? productsList(projects[project].products)
    : Object.keys(projects[project].products[0])[0],
components: ...
```
Will this work?